### DISCUSSION TITLE
Use unicode greek symbols and unicode-math package

### DISCUSSION THREAD

**Discussion by universemaster** [sleepymalc/VSCode-LaTeX-Inkscape#6] [Ideas]
I see you do not like `Tex Conceal` (prettify symbols), fair enough, me neither as it's a little too slow on a large document.

However, I also see from your `latex.hsnips` file that you also use the full latex commands for greek (and other) characters.

I find using the unicode characters makes my source a lot more readable, than having `\lambda` everywhere.

The [unicode-math](https://github.com/wspr/unicode-math) package allows you to use unicode symbols directly - see the [list](https://anorien.csc.warwick.ac.uk/mirrors/CTAN/macros/unicodetex/latex/unicode-math/unimath-symbols.pdf).

I particular like to use ∫ instead of `\int`.

I also like the use the subscript and superscript unicode [characters](https://github.com/wspr/unicode-math/blob/master/um-code-sscript.dtx) to again make my source more readable.

I don't need to worry about submitting to journals, but from what I read, use of unicode is becoming more accepted (and in my opinion should be accelerated!).

**Comment by sleepymalc** [comment]
Hmm, not sure whether this would help my workflow since in VS Code, the cross-reference feature (from pdf to source code by pressing `cmd + click`) can help me locate the equation I want to modify quite fast already, and I'm in particular comfortable looking at pure source code, so Unicode barely helps. And another excuse I'll make for myself is that since my VS Code is already slow (when integrating all other stuff like grammarly and etc.) I would like to minimize this kind of plug-in.

But yeah, if that unicode-math package is fast when compiling, I would like to try it out though I do not know a fast way to type out unicode tbh.
